### DISCUSSION TITLE
Question: How do you collect data about distribution ISO files

### DISCUSSION THREAD

**Discussion by STRRL** [ZJUSCT/mirror-front#14] [Q&A] [answered]
Hi! I am one of the maintainers of https://mirrors.ynu.edu.cn, and we are going to rebuild our mirror site webpage. We noticed this project, it's awesome.

I am curious about the feature that you could list ISO files, that's soo cool and very useful! 

<img width="1892" alt="image" src="https://user-images.githubusercontent.com/20221408/160222510-121c42d3-80fd-4e88-96d5-5fabc03bdde4.png">

And I noticed that the data about distribution ISO files are provided by the API https://mirrors.zju.edu.cn/api/mirrors

Could you tell me more about how do you collect this data? Tanks a lot! 

> We are using [tunasync](https://github.com/tuna/tunasync), and I do not find a similar feature from tunasync.

**Comment by iamNCJ** [comment, marked as answer]
The iso list is generated by our backend by scanning the files (which is not made public yet since we are still refactoring it), but the rules are basically derived from [tuna](https://github.com/tuna/mirror-web/blob/master/geninfo/genisolist.ini). You can refer to their awesome work. 🚀

**Comment by STRRL** [comment]
> 

Got that! Thanks!
